index.html

```diff
@@ -5,7 +5,7 @@
 <head>
     <meta charset="utf-8">
     <title>Skillcrush 105 - Advice For Newbies</title>
-    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="stylesheet" type="text/css" href="css/claudia-style.css" />
```

Comment:
It looks like you changed the stylesheet link here rather than in your personal advice file. We want this link to stay pointed to style.css.